lib/admin/class/module_visibility.dart

```diff
@@ -0,0 +1,69 @@
+class ModuleVisibilities {
+  ModuleVisibilities({
+    required this.root,
+    required this.allowedGroupIds,
+  });
+  late final String root;
+  late final List<dynamic> allowedGroupIds;
```

Comment:
Typing to dynamic is to avoid, it seems that this will be a list of String so type it as such

Reply:
For some reason, typing String here create problem with parsing from Json and provider then return an empty array

Reply:
Thus, it could better casting data when you are parsing json response and still typing it right